scripts/build.js

```diff
@@ -16,10 +16,14 @@ const pkg = require('../package.json');
 
 const dirRoot = join(__dirname, '..');
 
+const bundledBuilders = Object.keys(pkg.devDependencies)
+  .filter(d => d.startsWith('@now/'));
+
 async function createBuildersTarball() {
-  const builders = Object.keys(pkg.devDependencies)
-    .filter(d => d.startsWith('@now/'))
-    .map(d => `${d}@${pkg.devDependencies[d]}`);
+  const distTag = pkg.version.includes('-')
```

Comment:
I think you should use `npa()` here instead of includes/match. I already wrote a check to use canary/stable. See the function `getNpmVersion()` in `builder-cache.ts`.

Reply:
I think `semver.parse()` is actually more appropriate in this case. Updating.